src/lib/pickles/pickles.ml

```diff
@@ -3935,7 +2893,7 @@ module Make_str (_ : Wire_types.Concrete) = struct
 
         module Proof = (val p)
 
-        let example1 =
+        let _example1 =
```

Comment:
are these getting moved?

Reply:
I turned on warnings to help me deduplicate things; these values are unused, so these renamings squash the warnings.